lib/index.js

```diff
@@ -13,10 +13,21 @@ var handlers = require('./handlers')
 
 var own = {}.hasOwnProperty
 
+var deprecationWarningIssued = false
+
 // Factory to transform.
 function factory(tree, options) {
   var settings = options || {}
-  var dangerous = settings.allowDangerousHTML
+
+  // Issue a warning if the deprecated tag 'allowDangerousHTML' is used
+  if (settings.allowDangerousHTML !== undefined && !deprecationWarningIssued) {
+    deprecationWarningIssued = true
+    console.warn(
+      'Deprecation warning: `allowDangerousHTML` is a nonstandard option, use `allowDangerousHtml` instead'
```

Comment:
```suggestion
      'mdast-util-to-hast: deprecation: `allowDangerousHTML` is nonstandard, use `allowDangerousHtml` instead'
```

How about this? Bit shorter so it fits, but mostly to show where this message is coming from!

Reply:
Sure, seems good to me